Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_test.py

```diff
@@ -0,0 +1,102 @@
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_authenticate_user(requests_mock):
+    from Integrations.FireMonSecurityManager.FireMonSecurityManager import Client, authenticate_command
+
+    mock_response = util_load_json('Integrations/FireMonSecurityManager/test_data/get_authentication.json')
+    requests_mock.post('https://192.168.21.98/securitymanager/api/authentication/login', json=mock_response)
+
+    client = Client(
+        base_url='https://192.168.21.98',

```

Comment:
you can use here a test url, so it will not failed on validation.

Reply:
resolving as outdated